quarkus-extension/engine/deployment/src/test/java/org/camunda/bpm/quarkus/engine/test/config/CamundaEngineConfigFileTest.java

```diff
@@ -59,7 +59,7 @@ public void shouldLoadAllConfigProperties() throws SQLException {
     // assert engine config properties
     assertThat(configuration.isCmmnEnabled()).isFalse();
     assertThat(configuration.isDmnEnabled()).isFalse();
-    assertThat(configuration.getHistory()).isEqualTo("none");
+    assertThat(configuration.getHistory()).isEqualTo("full");
```

Comment:
This should be `none` since we set it in the `mixed-application.properties` file which has a higher priority.